src/page.js

```diff
@@ -61,13 +61,14 @@ export class Page {
   // Returns nothing.
   dispatch(data = {}) {
     const raw = this.recognize();
+    if (raw === void(0) || raw === null) {
+      throw new Error('<body> does not have a [data-page] attribute');
```

Comment:
@lucasmazza ah. thanks for the fast response. 

btw, the ci is failing and getting the same errors as i am locally. will try to look at that too.